core/meta/object-descriptor.js

```diff
@@ -363,13 +363,15 @@ var ObjectDescriptor = exports.ObjectDescriptor = Montage.specialize( /** @lends
 
     _handlePropertyDescriptorsRangeChange: {
         value: function (plus, minus, index) {
-            var i, n;
-            for (i = 0, n = minus.length; i < n; ++i) {
-                minus[i]._owner = null;
+            var descriptor, i, n;
```

Comment:
Unused `n`?

Reply:
Good catch! I have other not-yet-pushed changes related to that method and I will be be sure to remove that var.